### PR TITLE
Fix wrong measure unit in bandwidth control.

### PR DESCRIPTION
**Pull request by Zensey**
Resolves #3507

### CODE REVIEW

File: src/Components/BandwidthControl/BandwidthControl.tsx [hunk 1]
```diff
@@ -83,11 +86,11 @@ const BandwidthControl = ({
           onCancel()
           setShowConfirm(false)
           setChecked(turnedOn)
-          setBandwidth(bandwidthExt)
+          setBandwidth(bandwidthExt * kbitToMbit)
         }}
         onConfirm={async () => {
           if (typeof bandwidth === 'number') {
-            await onConfirm(checked, bandwidth)
+            await onConfirm(checked, bandwidth * mbitToKbit)
```
Comment (mdomasevicius) [inline src/Components/BandwidthControl/BandwidthControl.tsx:93]: `kbitToMbit` and `mbitToKbit` introduces complexity.
I would suggest you don't do mathematics in `BandwidthControl` at all and instead 
```javascript
          onConfirm={(isShaping: boolean, bandwidth: number) => {
              setState((cs) => ({ ...cs, isShaping: isShaping, bandwidth: bandwidth }))
              return restartServices(setTrafficShaping(isShaping, bandwidth * yourModifier))
            }}
```
As for `bandwidthExt * kbitToMbit` - just pass in value in Mbits. In other words keep `BandwidthControl` working with same units.

File: src/Pages/Authenticated/Dashboard/Services/GlobalServicesSettings.tsx [hunk 2]
```diff
@@ -87,9 +87,9 @@ const GlobalServicesSettings = ({ config, servicesInfos }: Props) => {
             message="This will restart all running services to take affect."
             turnedOn={state.isShaping}
             bandwidthExt={state.bandwidth}
-            onConfirm={(isShaping: boolean, bandwidth: number) => {
-              setState((cs) => ({ ...cs, isShaping: isShaping, bandwidth: bandwidth }))
-              return restartServices(setTrafficShaping(isShaping, bandwidth))
+            onConfirm={(isShaping: boolean, bandwidthKbit: number) => {
+              setState((cs) => ({ ...cs, isShaping: isShaping, bandwidth: bandwidthKbit }))
+              return restartServices(setTrafficShaping(isShaping, bandwidthKbit))
```
Comment (mdomasevicius) [inline src/Pages/Authenticated/Dashboard/Services/GlobalServicesSettings.tsx:92]: do math here :)
also rename argument of `setTrafficShaping(...)` - `bandwidth` -> `bandwidthKbit`

**Review by mdomasevicius — COMMENTED** [review]
Since this is public repo, let's keep up some standards towards meaningful commit messages :)
Please squish and summarize your work "
image

File: src/Components/BandwidthControl/BandwidthControl.tsx (hunk 1 above)
Comment (Zensey) [reply src/Components/BandwidthControl/BandwidthControl.tsx]: ok

File: src/Pages/Authenticated/Dashboard/Services/GlobalServicesSettings.tsx (hunk 2 above)
Comment (Zensey) [reply src/Pages/Authenticated/Dashboard/Services/GlobalServicesSettings.tsx]: ok

File: src/Pages/Authenticated/Dashboard/Services/GlobalServicesSettings.tsx [hunk 3]
```diff
@@ -86,10 +86,10 @@ const GlobalServicesSettings = ({ config, servicesInfos }: Props) => {
           <BandwidthControl
             message="This will restart all running services to take affect."
             turnedOn={state.isShaping}
-            bandwidthExt={state.bandwidth}
+            bandwidthExt={state.bandwidth / 1000}
```
Comment (chompomonim) [inline src/Pages/Authenticated/Dashboard/Services/GlobalServicesSettings.tsx:89]: If this is in kbps then maybe multiply by 1024?

Comment (Zensey) [reply src/Pages/Authenticated/Dashboard/Services/GlobalServicesSettings.tsx:89]: No.
In bit rates the abbreviations are as follows:

1 kbps = 1,000 bits per second
1 Mbps = 1,000,000 bits per second.
1 Gbps = 1,000,000,000 bits per second.

Comment (mdomasevicius) [reply src/Pages/Authenticated/Dashboard/Services/GlobalServicesSettings.tsx:89]: @Zensey is correct, AFAIK we introduced two concepts KB vs KiB - one is metric (kilo as in 1000) other is binary (1000 vs 1024)

**Review by chompomonim — APPROVED** [review]

**Pull request merged**